### DISCUSSION TITLE
Add tags to categorize links inside a crate

### DISCUSSION THREAD

**Discussion by mynameismon** [WebCrateApp/webcrate#24] [Ideas]
Tags are a great way to identify and group information, and is pretty much a must need for any organisational software.

**Comment by BetaHuhn** [comment]
Hey, thanks for opening this discussion!

You can group links together with crates. How would you use tags in addition to this?

(Selecting a crate when adding a link via the extension is coming soon).

**Comment by mynameismon** [comment]
For example, I have a crate storing links related to Maths and Physics. I would want to categorise them by type, so for example, one is a textbook, the other is a lecture series, the third is an article, so on and so forth.

**Reply by BetaHuhn** [reply]
Okay, makes sense! Will add it to the [list of feature requests](https://github.com/WebCrateApp/webcrate/projects/1)!
